src/schedule/zephyr_dp_schedule.c

```diff
@@ -253,7 +252,6 @@ void scheduler_dp_ll_tick(void *receiver_data, enum notify_id event_type, void *
 				/* set a deadline for given num of ticks, starting now */
 				k_thread_deadline_set(pdata->thread_id,
 						      pdata->deadline_clock_ticks);
-				pdata->ll_cycles_to_deadline = pdata->deadline_ll_cycles;
```

Comment:
I can see, that this re-arming is useless, but it seems it should be harmless, since the `dp_startup_delay` flag is only set to `true` once during task initialisation? What am I missing?

Reply:
flag may never be set to false if re-arming occurs too early. That hurts.